### ISSUE TITLE
WARNING because Requires.jl in Julia 0.5

### ISSUE THREAD

**Issue by diegozea** [davidavdav/ROCAnalysis.jl#3]
I see this warning using Julia 0.5.0:

```
julia> using ROCAnalysis
WARNING: Method definition require(Symbol) in module Base at loading.jl:345 overwritten in module Main at /home/dzea/.julia/v0.5/Requires/src/require.jl:12.
```

Can `@require` be replaced with something like [**this**](https://github.com/diegozea/PairwiseListMatrices.jl/blob/master/src/PairwiseListMatrices.jl#L6)?

Best,

**Comment by davidavdav** [comment]
Perhaps---it all seems to be not very pretty.  I thought `require()` was solving an issue for me here, but it seems it doesn't anymore.  Is this a bug of `Requires`?

I not very happy with plotting in Julia.  As a package developer, I don't want to have to choose which plotting package a user uses.  I'd like an abstract interface---but hey, that is not trivial. 

I've tried several plotting packages, and I haven't found anything that gives me the quality that I am used to from R.

**Comment by ValdarT** [comment]
> As a package developer, I don't want to have to choose which plotting package a user uses. I'd like an abstract interface

Doesn't Plots.jl and its 'recipes' solve exactly this problem (https://juliaplots.github.io/)?

I hope I am not misunderstanding anything.

**Comment by davidavdav** [comment]
Ehm, yes, that look fancy.  I hope it also support simple 2D graph plotting in a nice way.

**Comment by ValdarT** [comment]
Looks nice enough to me: https://github.com/JuliaML/MLPlots.jl

**Comment by davidavdav** [comment]
I'll keep this open as a separate issue for now.  I would like to move to a uniform plotting interface because all the various plotting packages in Julia drive me nuts.
